mosquitto/DOCS.md

```diff
@@ -97,7 +97,7 @@ See the following links for more information:
 - [Mosquitto topic restrictions](http://www.steves-internet-guide.com/topic-restriction-mosquitto-configuration/)
 - [Mosquitto.conf man page](https://mosquitto.org/man/mosquitto-conf-5.html)
 
-Add the following configuration to enable **unrestricted** access to all topics.
+Add the following configuration to enable **unrestricted** access to all topics for `[YOUR_MQTT_USER]`. Once you enable MQTT ACLs, home assistant will no longer have permission to publish or subscribe to the topics that it needs to. It is suggested that you add the `homeassistant` and `addons` user to the ACLs as demonstrated below.
```

Comment:
```suggestion
Add the following configuration to enable **unrestricted** access to all topics for `[YOUR_MQTT_USER]`.

**Note:** Home Assistant expects the users `homeassistant` and `addons` to have unrestricted readwrite access to all topics. If you choose to enable ACLs, you should grant these users these permissions as demonstrated below. Otherwise you will run into issues.
```